docs/Wiki/ProductAndTestProjects.md

```diff
@@ -1,29 +1,62 @@
 
 ### Differences between analysis of test projects and product projects
 
-The Sonar Scanner for MSBuild ("S4MSB") analyses MSBuild projects containing product code differently from projects containing test code.
+The S4NET analyses MSBuild projects containing product code differently from projects containing test code.
+
+#### Analysis of product projects
+
+* analysis rules will be run against product projects, and the issues raised will be uploaded to SonarQube/SonarCloud.
+* only rules related to product code will execute.
+* metrics and Lines of Code ("LOC") limit for commercial versions of SonarQube or private SonarCloud projects are calculated.
 
 #### Analysis of test projects
-* analysis rules are not run against test projects i.e. no issues will be reported to SonarQube/SonarCloud. This is the case even if the test project references third-party NuGet analyzer packages - those analyzers will not be executed.
+
+* analysis rules will be run against test projects, unless excluded (see below), and the issues raised will be uploaded to SonarQube/SonarCloud.
+* only rules related to test code will execute.
 * test projects do not count towards to the Lines of Code ("LOC") limit for commercial versions of SonarQube or private SonarCloud projects.
 * metrics are not calculated for test projects, although syntax colourisation and symbol highlighting are supported.
 
-#### Analysis of product projects
-* analysis rules will be run against product projects, and the issues raised will be uploaded to SonarQube/SonarCloud.
-* the `SonarAnalyzer.CSharp`/`VB` rules will be executed even if those analyzers are not referenced in the MSBuild project (S4MSB will add references to those analyzers on the fly during the build). The set of rules that is executed is determined by the Quality Profile for the Sonar project in SonarQube/SonarCloud.
-* any third-party analyzers referenced by the MSBuild project will be executed as part of the build (e.g. via NuGet package references) . Issues from those analyzers will be uploaded to SonarQube/SonarCloud as [External Issues](https://sonarcloud.io/documentation/analysis/external-issues/). You can configure the third-party analyzer rules as normal using a `Ruleset`. During the build, S4MSB will merge your custom ruleset with a ruleset generated from the Quality Profile. In the event of a conflict, the settings in the generated ruleset take precedence.
+#### Analysis of excluded test projects
+
+* test projects are excluded from analysis on SonarQube up to version 8.8
+* test projects can be excluded from analysis on SonarCloud and SonarQube from version 8.9 by adding `/d:sonar.dotnet.excludeTestProjects=true` S4NET parameter.
+* analysis rules are not run against excluded test projects i.e. no issues will be reported to SonarQube/SonarCloud. This is the case even if the test project references third-party NuGet analyzer packages - those analyzers will not be executed.
```

Comment:
```suggestion
* if test projects are excluded, no analyzers will be executed on any test project, i.e. no issues will be reported to SonarQube/SonarCloud. This is the case even if the test project references third-party NuGet analyzer packages - those analyzers will not be executed.
```

Reply:
I find the proposed version confusing. We're in `Analysis of excluded test projects` section. 
The confusing part:
`if test projects are excluded` - If? Is there a way to not-exclude some when I exclude them? 

I'll put `all test projects` on the previous points to make it more explicit.